lib/src/migrators/division.dart

```diff
@@ -65,6 +73,49 @@ class _DivisionMigrationVisitor extends MigrationVisitor {
     }
   }
 
+  /// Allows division within a function call's arguments, with special handling
+  /// for new-syntax color functions.
+  @override
+  void visitFunctionExpression(FunctionExpression node) {
+    visitInterpolation(node.name);
+    if (node.name.asPlain == "rgb" ||
+        node.name.asPlain == "rgba" ||
+        node.name.asPlain == "hsl" ||
+        node.name.asPlain == "hsla") {
```

Comment:
It would be great to remove some of this nesting... maybe factor this out into `bool _tryColorFunction(FunctionExpression node)` that can just `return false` every time it determines that the function doesn't need special handling?